android/build.gradle

```diff
@@ -2,10 +2,10 @@ group 'com.example.flutter_sms'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.10'
     repositories {
         google()
-        jcenter()
```

Comment:
Is there a reason to switch to mavenCentral? This SO post suggests jcenter is more secure and offers better CDN performance.

https://stackoverflow.com/questions/50726435/difference-among-mavencentral-jcenter-and-mavenlocal

Reply:
Here is an official note from android.
https://developer.android.com/studio/build/jcenter-migration

Jcenter is being deprecated in favour of maven central

Reply:
Oof, thanks for the heads up.  This wound up being the root cause of some issues with some of my transient dependencies.